### PR TITLE
Update dependencies

### LINKED ISSUE #109: Update eslint v4.9
latest: https://eslint.org/blog/2017/10/eslint-v4.9.0-released

**new rules**

- [multiline-comment-style](https://eslint.org/docs/rules/multiline-comment-style)
  - いらなそう、コメントのスタイルそんなに縛らんでもええやろ
  - `multiline-comment-style: 0`
- [lines-between-class-members](https://eslint.org/docs/rules/lines-between-class-members)
  - 読みやすそうかなーとは思うからありでもよさそうやけどどうやろ

### PR DESCRIPTION
**Pull request by kanufy**
fix #109 
refs #107 

eslint-plugin-node もあがってたので一緒にあげてます

### CODE REVIEW

File: config/eslintrc_core.js [hunk 1]
```diff
@@ -241,8 +241,9 @@ module.exports = {
             'after': true,
         }],
         'linebreak-style': [2, 'unix'],
-        'lines-around-comment': 0, // https://eslint.org/docs/rules/lines-around-comment
         'line-comment-position': 0, // This is needless. https://eslint.org/docs/rules/line-comment-position
+        'lines-around-comment': 0, // https://eslint.org/docs/rules/lines-around-comment
```
Comment (nekoya) [inline config/eslintrc_core.js]: `lines-around-comment`より`line-comment-position`の方が先だったか

https://eslint.org/docs/rules/

**Comment by nekoya** [comment]
@popuko r+

**Comment by popuko** [comment]
:pushpin: Commit 0a774e492c7c64d7a42de10f7c231c9a0cd3da78 has been approved by `nekoya`

**Comment by popuko** [comment]
:hourglass: 0a774e492c7c64d7a42de10f7c231c9a0cd3da78 has been merged into the auto branch e6a87039a5626649d3292982aee3fe8c24926855

**Comment by popuko** [comment]
:tada: The result of what tried to merge this pull request is `success`.

* [The Travis CI build passed](https://travis-ci.org/voyagegroup/eslint-config-fluct/builds/291334836?utm_source=github_status&utm_medium=notification)

**Pull request merged**